p_for_python.py

```diff
@@ -0,0 +1,7 @@
+def get_num():
+    list1 = [chr(num) for num in range(ord('A'), ord('['))]
+    list2 = [chr(num) for num in range(ord('a'), ord('{'))]
+    return list1, list2
```

Comment:
This is not a list, its a tuple of two list. Add return type and pycharm will scream that you are returning the wrong type

Reply:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/